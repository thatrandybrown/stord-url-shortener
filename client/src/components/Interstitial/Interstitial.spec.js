import { render } from "@testing-library/react";
import Loader from "./";

test("renders the Interstitial Component", () => {
  const { baseElement } = render(<Loader />);
  expect(baseElement.firstChild).toMatchSnapshot();
});
