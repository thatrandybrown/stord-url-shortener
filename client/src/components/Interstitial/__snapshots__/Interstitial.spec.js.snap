// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Interstitial Component 1`] = `
<div
  id="interstitial-root"
>
  <div>
    <svg
      height="64"
      version="1"
      viewBox="0 0 128 128"
      width="64"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g>
        <path
          d="M78.75 16.18V1.56a64.1 64.1 0 0147.7 47.7H111.8a49.98 49.98 0 00-33.07-33.08zM16.43 49.25H1.8a64.1 64.1 0 0147.7-47.7V16.2a49.98 49.98 0 00-33.07 33.07zm33.07 62.32v14.62A64.1 64.1 0 011.8 78.5h14.63a49.98 49.98 0 0033.07 33.07zm62.32-33.07h14.62a64.1 64.1 0 01-47.7 47.7v-14.63a49.98 49.98 0 0033.08-33.07z"
          fill="#b011b0"
        />
        <animatetransform
          attributeName="transform"
          dur="400ms"
          from="0 64 64"
          repeatCount="indefinite"
          to="-90 64 64"
          type="rotate"
        />
      </g>
    </svg>
  </div>
</div>
`;
